db/db.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
```

Comment:
Mongoose 5.5.7 should be able to support its native promise without bluebird. :)

Reply:
I just tested it and sure enough it works.